README.md

```diff
@@ -36,13 +36,20 @@ To run a dockerized version of our postgres database locally, run `make database
 ## Development recommendations
 
 ### First time setup
-When you first are setting up the application, copy the file titled `.env_example` to be `.env`, and configure the missing enviornment variables. To add the bot account to your server, you can follow the [discord.py instructions](https://discordpy.readthedocs.io/en/stable/discord.html). The server that you use for development also will need to have a channel where the bot will output logging messages, and will need to have the following roles set up
-* Exiled
-* Verified
+We recomment getting a ]virtual environment](https://docs.python.org/3/library/venv.html) set up for python before you start development. All required packages to run the application are defined in `requirements.txt`
```

Comment:
`recomment` -> `recommend` and open bracket is facing the wrong way

Reply:
guh untypable